shuup/apps/provides.py

```diff
@@ -170,61 +171,64 @@ def override_provides(category, spec_list):
         _provide_specs[category] = old_provides
 
 
-def load_module(setting_name, provide_category):
+def load_module(configuration_name, provide_category):
```

Comment:
it is not a good idea to touch on this function. I recall we are using this in several places in addons and projects. I would rather create a new  function to load from a configuration or create a new parameter to control this.

Reply:
Ok, I'm going to separate them, but I think is safer the way I implemented, as load_module looks first in the configuration and it falls back to settings. Any add-on using load_module will still work. If there is an add-on relying on the settings that we moved, will no longer work. So I must identify all the cases.